app/views/spree/api/affirm/payload.json.jbuilder

```diff
@@ -1,6 +1,6 @@
-json.config do
-  json.financial_product_key Spree::Config.get(:affirm_financial_product_key)
-end
+# json.config do
```

Comment:
We dont commit commented out code. We commit comments only when they add meaningful context to the code.